Source/Scene/Camera.js

```diff
@@ -2972,6 +2972,7 @@ function getPickRayOrthographic(camera, windowPosition, result) {
  * @param {Cartesian2} windowPosition The x and y coordinates of a pixel.
  * @param {Ray} [result] The object onto which to store the result.
  * @returns {Ray} Returns the {@link Cartesian3} position and direction of the ray.
+ * If the Scene is not rendered or the pickRay cannot be determined, returns undefined.
```

Comment:
We can actually do this directly in the jsDoc and don't need to add it a sperate bullet:

```suggestion
 * @returns {Ray|undefined} Returns the {@link Cartesian3} position and direction of the ray, or undefined if the pick ray cannot be determined.
```

It's important to update the return type since we generate TypeScript definitions from our jsDoc comments.